spk/ffsync/Makefile

```diff
@@ -40,6 +40,7 @@ include ../../mk/spksrc.spk.mk
 ffsync_extra_install:
 	install -m 755 -d $(STAGING_DIR)/var
 	install -m 644 src/ffsync.ini $(STAGING_DIR)/var/
+	install -m 644 src/requirements.txt $(STAGING_DIR)/share/wheelhouse/requirements.txt
```

Comment:
Otherwise the `sed` operation below was failing. To be honest it's either we remove that line **and** the below line (the `sed` one). Or we keep that line or at least I need to know where that `requirements.txt` file would be supposed to be coming from.